cypress/integration/Pages/HomePage.js

```diff
@@ -0,0 +1,52 @@
+class HomePage {
+
+    visit(){
+        return cy.visit('https://usps.returns-portal-admin.apac-preprod.doddle.tech/merchants/request')
```

Comment:
try using the base URL here. If you set the base URL code will look like this: cy.visit('/merchants/request')
```suggestion
        return cy.visit('/merchants/request')
```